helpers/chatbot/intentResponse.js

```diff
@@ -0,0 +1,99 @@
+// Function to manage custom intents are defined here
+module.exports = {
+    CheckMalariaSymptoms: function(parameters, done) {
+
+        const symptomScores = {
+            chills: 2,
+            fatigue: 2,
+            fever: 2,
+            malaise: 2,
+            shivering: 2,
+            sweating: 2,
+            headache: 1,
+            nausea: 1,
+            'pain in muscles': 1,
+            'faster heart rate': 1
+        };
+
+        const symptoms = parameters.Symptoms;
+        let length = symptoms.length;
+        let malariaRisk = 0;
+
+        symptoms.forEach(function(symptom) {
+            malariaRisk += symptomScores[symptom];
+            let reply;
+            if (--length === 0) {
+                if (malariaRisk > 6) {
+                    reply = 'You seem to have quite a lot of symptoms related to malaria. Malaria can be a threat to life, please visit your doctor Immediately.';
```

Comment:
Use node specific english.json

Reply:
@Raxerz Currently DialogFlow is trained only in the English language (although it supports other languages as well). Unless we train it for another language, this might not be useful.

Reply:
Yes, but even then for consistency purposes.. it is good to use english.json. It's good practice not to have hard coded strings, not just for multi language purpose.